install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -1189,8 +1189,12 @@ data:
   annotate-k8s-node: "true"
 {{- end }}
 
-  k8s-client-qps: {{ .Values.k8sClientRateLimit.qps | default $defaultK8sClientQPS | quote}}
-  k8s-client-burst: {{ .Values.k8sClientRateLimit.burst | default $defaultK8sClientBurst | quote }}
+{{- if .Values.k8sClientRateLimit.qps }}
```

Comment:
Might be useful adding a comment here reporting what you said in PR description:

```
Configuring a custom rate limit via Helm will still set the rate limit for both Cilium Agent and Cilium Operator. If a separate custom rate limit is desired for the Operator, .Values.operator.extraArgs can be used to supply the flags directly to the deployment.
```

Reply:
Alternatively, adding that to the OperatorK8sClientQPS{Limit,Burst} docstrings.
What is concerning me is the counterintuitive behavior of having two separate options with different defaults but the same knob to tweak them (although I understand the reason).

Reply:
I like your suggestion of moving everything under `operator/`. If we do that, we could use different names for the options and separate helm values. I think this makes the most sense. WDYT?